src/features/board/components/board-issue-drawer.tsx

```
@@ -913,30 +915,53 @@ const BlockingSection = ({
+										<div className="text-xs text-muted-foreground mt-1 line-clamp-2">
+											{reasoning}
+										</div>
+									)}
+									{resolutionSteps && resolutionSteps.length > 0 && (
+										<div className="mt-2 rounded-md border bg-muted/20 px-2 py-2">
+											<div className="text-[11px] font-medium text-muted-foreground">
+												Resolution steps
+											</div>
+											<ol className="mt-1 list-decimal pl-4 space-y-0.5">
+												{resolutionSteps.slice(0, 7).map((step, idx) => (
+													<li className="text-xs text-foreground/90" key={idx}>
```

Biome (2.4.14)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)